blog/02-18-2023-how-to-use-hardware-tools-with-the-celo-blockchain/how-to-use-hardware-tools-with-the-celo-blockchain.md

```diff
@@ -0,0 +1,125 @@
+
+With hardware technology, devices used on the blockchain are designed for a specific purpose. Such devices can be optimized for security and speed thanks to their specific architecture which gets rid of loopholes common to most general-purpose systems.
+
+Hardware devices isolate the vital computations and data into an IP core that gets separated from the rest of the system. For example, with hardware wallets, cryptographic keys can get isolated from the rest of the system into their memory. This memory can be accessed only by the cryptographic IP cores that require these keys. That upgrade is a massive security improvement as it effectively wades off most of the attacks discussed above. 
+
+So, in rare cases where the security of frontend programs or the operating system running on the processor of the main system gets compromised, the attacker still has no access to the keys. Should the IP core itself have a bug that allows a buffer overflow type of attack, the keys stay safe as they are duly isolated inside the IP core. 
+
+# Hardware and the Celo Blockchain: Current Applications
+
+The current applications of hardware and blockchain generally (with Celo as a standard example) are beyond providing payment functionalities. Here are some of the most prominent applications of the intersection of these two technologies in our present world:
+
+- ## Crypto Assets Management and Security
+
+At the start, the goal of the blockchain was to provide a robust payment system void of ownership but structured to deliver on many grounds. This remains the most important use of blockchain networks. For context, the Celo blockchain aims to provide payment solutions that are seamless for all and create an avenue for prosperity for everyone.
+
+One of the many ways they do this is by enabling the use of several, but not all hardware wallets on their blockchain to encrypt and secure your digital assets using keys. With this hardware technology application, there is more security and assurance for these assets as they are off the online space and no longer susceptible to cyberattacks. The good news is, there is an ample list of hardware wallets compatible with the Celo blockchain network 
+
+- ## Decentralized Storage
+
+Hardware technology delivers a step up from what blockchain provides with DeFI by enabling decentralized storage. In a decentralized storage system, as opposed to storing all the data in a centralized server, data storage gets distributed into various chunks. These chunks get stored inside various nodes of a peer-to-peer (P2P) network. 
+
+One benefit of using such storage is that they provide the same benefits as a decentralized web. They are more secure, encourage privacy, are cost-effective, and maintain no single point of failure. Also, on Celo, these decentralized storages help decentralized applications (dApps) built on the network to permanently or temporarily store data. 
+
+- ## Sensor Nodes for Oracle Applications on the Blockchain
```

Comment:
This is awesome but I'm guessing this is intended to be a point under the previous heading,
If that is the case you can make it a triple hashtag and also